src/wp-includes/js/media-views.js

```diff
@@ -4409,13 +4409,20 @@
 		/**
 		 * Gets all the tabbable elements.
 		 *
-		 * @since 5.3.0
+		 * @since CP-2.1.0
 		 *
-		 * @return {Object} A jQuery collection of tabbable elements.
+		 * @return {Array} A JavaScript array of tabbable elements.
 		 */
 		getTabbables: function() {
+			var tabbables = [ ...document.querySelector( '.media-modal' ).querySelectorAll( 'a[href], button, input, textarea, select, [tabindex]' ) ];
+
 			// Skip the file input added by Plupload.
-			return this.$( ':tabbable' ).not( '.moxie-shim input[type="file"]' );
+			for ( var i = 0, n = tabbables.length; i < n; i++ ) {
+				if ( tabbables[i].tagName === 'input' && tabbables[i].className.includes( 'moxie-shim' ) && tababbles[i].type === 'file' ) {
```

Comment:
Typo here in the last comparison - `tababbles[i].type === 'file'` noted that 'tababbles' is misspelled.

Reply:
I have pushed a change that should resolve this.

Reply:
Thanks for spotting and resolving that!